apache/superset
# fix: tags permissions error message

### SUMMARY


Error message in the case of wrong tags permissions is confusing.
<img width="596" alt="Screenshot 2023-10-04 at 12 23 09" src="https://github.com/apache/superset/assets/3982146/53a05570-3cda-4667-85c4-d47f214a2395">

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS


1. Enable `TAGGING_SYSTEM` feature flag
2. Remove `can read on Tag` permission from your role.
3. Open the list of dashboards and try to see the list of tags

Expected result:

The error message should be `You do not have permission to read tags`

### ADDITIONAL INFORMATION
